packages/frontend/web/src/components/ThemeSwitcher.tsx

```diff
@@ -1,19 +1,12 @@
-import { useEffect, useState } from 'react';
+/* import { useEffect, useState } from 'react'; */
+import MoonIcon from './icons/MoonIcon';
+import SunIcon from './icons/SunIcon';
 
 export default function ThemeSwitcher() {
-  const [isDarkMode, setIsDarkMode] = useState(false);
-
-  useEffect(() => {
-    document.documentElement.classList.toggle('dark', isDarkMode);
-  }, [isDarkMode]);
-
-  const toggleDarkMode = () => {
-    setIsDarkMode((prevMode) => !prevMode);
-  };
-
   return (
-    <button type='button' onClick={toggleDarkMode}>
-      {isDarkMode ? 'Light Mode' : 'Dark Mode'}
-    </button>
+    <>
+      <MoonIcon className='text-secondary scale-moon h-4 w-4 md:h-5 md:w-5' />
+      <SunIcon className='text-secondary scale-sun absolute h-4 w-4 md:h-5 md:w-5' />
```

Comment:
Les `scale-` en mode variables sont un peu overkill mais why not
(`scale-0 dark:scale-1` et `scale-1 dark:scale-0` auraient suffit)